test/fixtures/full_config.yml

```diff
@@ -2353,6 +2378,12 @@ Style/ArrayCoercion:
   Safe: false
   Enabled: false
   VersionAdded: '0.88'
+Style/ArrayFirstLast:
+  Description: Use `arr.first` and `arr.last` instead of `arr[0]` and `arr[-1]`.
+  Reference: "#first-and-last"
+  Enabled: false
+  VersionAdded: '1.58'
+  Safe: false
```

Comment:
I kept this disabled, as I don't see it being worth it given how many false positives it would have.